### PR TITLE
Fix documentation for rust packages

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
# Description
- fix documentation output for rust.
image

## Check list
- [ ] Related issue / work item is attached
- [x] Tests are written (if applicable)
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 1]
```diff
@@ -115,7 +114,9 @@ $markdown += New-MDList -Style Unordered -Lines @(
     (Get-GoogleCloudSDKVersion)
 )
 
-$markdown += New-MDHeader "Rust packages:" -Level 3
+$markdown += New-MDHeader "Rust" -Level 3
+$markdown += New-MDParagraph -Lines "Version: $(Get-RustVersion)"
+$markdown += New-MDHeader "Packages:" -Level 3
```
Comment (maxim-lobanov) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:119]: We don't need `:` here

Comment (maxim-lobanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:119]: Also I think `Packages` section should have level higher than title

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 2]
```diff
@@ -115,7 +114,9 @@ $markdown += New-MDList -Style Unordered -Lines @(
     (Get-GoogleCloudSDKVersion)
 )
 
-$markdown += New-MDHeader "Rust packages:" -Level 3
+$markdown += New-MDHeader "Rust" -Level 3
+$markdown += New-MDParagraph -Lines "Version: $(Get-RustVersion)"
```
Comment (maxim-lobanov) [inline images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1:118]: Let's replace `Version:` on `Rust` like we have done for MacOS

Comment (dmitry-shibanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Done.

File: images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1 (hunk 1 above)
Comment (dmitry-shibanov) [reply images/win/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Done.

**Pull request merged**